modules/markup/renderer.go

```diff
@@ -81,6 +81,7 @@ type Renderer interface {
 	Extensions() []string
 	NeedPostProcess() bool
 	SanitizerRules() []setting.MarkupSanitizerRule
+	SanitizeDisabled() bool
 	Render(ctx *RenderContext, input io.Reader, output io.Writer) error
```

Comment:
here, `SanitizeDisabled`  vs `SanitizerRules` , I think `Sanitizer` is the correct name.